Gruntfile.js

```diff
@@ -154,15 +154,58 @@ module.exports = function (grunt) {
     preloadSpreadsheet: {
       'v0.2': 'https://docs.google.com/spreadsheets/d/1ughCy983eK-SPIcDYPsjOitVZzY10WdI2MGGrmxzxF4/pubhtml',
     },
-    buildSwf: {
-      dev: {
-        swfPath: "./.tmp/storage.swf",
-        asPath: "./jsflash/dev/Storage.as"
+
+    mxmlc: {
+        options: {
+          // Task-specific options go here.
+        },
+        dev: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/dev/Storage.as']
+          }
+        },
+        prod: {
+          files:{
+            './.tmp/storage.swf' : ['./jsflash/prod/Storage.as']
```

Comment:
okay..  i'm still not entirely clear on the whole dist vs .tmp thing works.. im still learning about grunt/bower/npm etc...

 i'll try and be more careful if i make more build related changes..